apps/sentry/src/app/install/action.tsx

```diff
@@ -15,6 +16,7 @@ export type FormState = {
   redirectUrl?: string;
   errors?: {
     token?: string[] | undefined;
+    sourceOrganizationId?: string[] | undefined;
```

Comment:
i think we are getting organization slug an not the organization id. Can you confirm this?

Reply:
Done